frappe/public/js/frappe/list/list_view.js

```diff
@@ -514,7 +514,7 @@ frappe.views.ListView = class ListView extends frappe.views.BaseList {
 
 	render_skeleton() {
 		const $row = this.get_list_row_html_skeleton(
-			'<div><input type="checkbox" /></div>'
+			'<div><input type="checkbox" style="margin-left: 15px;"/></div>'
```

Comment:
let's avoid inline CSS to keep it overridable? 

you can create a different CSS class and move this.